listenbrainz/db/user_relationship.py

```diff
@@ -0,0 +1,74 @@
+# listenbrainz-server - Server for the ListenBrainz project.
+#
+# Copyright (C) 2020 Param Singh <iliekcomputers@gmail.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+from listenbrainz import db
+from listenbrainz.db.exceptions import DatabaseException
+
+import sqlalchemy
+
+VALID_RELATIONSHIP_TYPES = (
+    'follow',
+)
+
+
+def insert(user_0: int, user_1: int, relationship_type: str) -> None:
```

Comment:
Do we have an official policy on docstrings? It is quite clear what the code does and a docstring isn't really needed, IMHO. When do we require them and when not?